app/src/main/java/io/intrepid/contest/screens/contestresults/ContestResultsContract.java

```diff
@@ -0,0 +1,20 @@
+package io.intrepid.contest.screens.contestresults;
+
+import java.util.List;
+
+import io.intrepid.contest.base.BaseContract;
+import io.intrepid.contest.models.RankedEntryResult;
+
+class ContestResultsContract {
+    interface View extends BaseContract.View {
+        void setupAdapter();
```

Comment:
True, for this case it could be considered a View task. I just did the same as `SelectContacts` in which case I needed a variable from the presenter.